relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -532,6 +562,43 @@ fn scrub_function(string: &str) -> Option<String> {
     Some(FUNCTION_NORMALIZER_REGEX.replace_all(string, "*").into())
 }
 
+fn scrub_mongodb_query(query: &str, command: &str, collection: &str) -> Option<String> {
+    let original: Value = match serde_json::from_str(query) {
+        Ok(v) => v,
+        Err(_) => return None,
+    };
+
+    let mut scrubbed_map: Map<String, Value> = Map::new();
+    original.as_object()?.iter().for_each(|(key, value)| {
+        scrubbed_map.insert(key.clone(), scrub_mongodb_leaf_nodes(value));
+    });
+
+    scrubbed_map.insert(command.to_owned(), Value::String(collection.to_owned()));
+
+    let scrubbed_query = Value::Object(scrubbed_map).to_string();
+    Some(scrubbed_query)
+}
+
+fn scrub_mongodb_leaf_nodes(value: &Value) -> Value {
+    match value {
+        Value::Object(map) => {
+            let mut scrubbed_map = Map::new();
+            map.iter().for_each(|(key, value)| {
+                scrubbed_map.insert(key.clone(), scrub_mongodb_leaf_nodes(value));
```

Comment:
Do we need a max recursion limit/depth here?

Reply:
Added a conservative limit (too deep seems like it would invite cardinality issues).